src/m365/spfx/commands/project/project-upgrade.ts

```diff
@@ -719,7 +719,7 @@ ${f.resolution}
     }
 
     if (args.options.shell) {
-      if (['bash', 'powershell', 'logger'].indexOf(args.options.shell) < 0) {
+      if (['bash', 'powershell', 'cmd'].indexOf(args.options.shell) < 0) {
```

Comment:
To avoid such inconsistencies in the future, let's move it to an array that we can reuse across autocomplete, validation and validation error message